docs/Displaying a User Interface/Template Images/index.md

```diff
@@ -94,6 +94,15 @@ sdlManager.getScreenManager().setPrimaryGraphic(null);
 ```
 !@
 
+### Overwriting Images
+When a file is to be uploaded to the module, the library checks if a file with the same name has already been uploaded to module and skips the upload if it can. For cases where an image by the same name needs to be re-uploaded, the @![iOS]`SDLArtwork` / `SDLFile`'s !@@![android, javaSE, javaEE, javascript]`SdlArtwork` \ `SdlFile`'s!@ `overwrite` property should be used. Setting `overwrite` to `true` before passing the image to a @![iOS]`SDLScreenManager`!@@![android, javaSE, javaEE, javascript]`ScreenManager`!@ method such as @![iOS]`primaryGraphic` and `secondaryGraphic`!@@![android, javaSE, javaEE, javascript]`setPrimaryGraphic()` and `setSecondaryGraphic()`!@ will force the image to be re-uploaded. This includes methods such as @![iOS]`preloadChoices:withCompletionHandler:`!@@![android, javaSE, javaEE, javascript]`preloadChoices()`!@ where the arguments passed in contain images.
```

Comment:
Suggesting to use forward slash in both cases
```suggestion
When a file is to be uploaded to the module, the library checks if a file with the same name has already been uploaded to module and skips the upload if it can. For cases where an image by the same name needs to be re-uploaded, the @![iOS]`SDLArtwork` / `SDLFile`'s !@@![android, javaSE, javaEE, javascript]`SdlArtwork` / `SdlFile`'s!@ `overwrite` property should be used. Setting `overwrite` to `true` before passing the image to a @![iOS]`SDLScreenManager`!@@![android, javaSE, javaEE, javascript]`ScreenManager`!@ method such as @![iOS]`primaryGraphic` and `secondaryGraphic`!@@![android, javaSE, javaEE, javascript]`setPrimaryGraphic()` and `setSecondaryGraphic()`!@ will force the image to be re-uploaded. This includes methods such as @![iOS]`preloadChoices:withCompletionHandler:`!@@![android, javaSE, javaEE, javascript]`preloadChoices()`!@ where the arguments passed in contain images.
```